app/models/agents/website_agent.rb

```diff
@@ -105,6 +106,10 @@ class WebsiteAgent < Agent
           * `status`: HTTP status as integer. (Almost always 200)
 
           * `headers`: Response headers; for example, `{{ _response_.headers.Content-Type }}` expands to the value of the Content-Type header.  Keys are insensitive to cases and -/_.
+
+      # Ordering Events
```

Comment:
We may need to add some JS that folds sections of the description since it's getting so long. :wink: